.pre-commit-config.yaml

```diff
@@ -53,14 +35,22 @@ repos:
         language: system
         types: [python]
         files: tests
-  - repo: local
-    hooks:
       - id: mypy
         name: mypy
         entry: ./script/run-pre-commit.sh mypy
         language: script
         types: [python]
         require_serial: true
-        #        files: ^(pytradfri|examples|tests)/.+\.py$
-        # DO NOT ADD examples/tests until files have been updated.
-        files: ^(pytradfri)/.+\.py$
+        files: ^(pytradfri|examples|tests)/.+\.py$
```

Comment:
Does mypy pass for all examples and tests? In that case we should activate it in tox too.

Reply:
I believe so:
```
(.venv) vscode ➜ /workspaces/pytradfri (enable-pre-commit) $ mypy pytradfri
Success: no issues found in 27 source files
(.venv) vscode ➜ /workspaces/pytradfri (enable-pre-commit) $ mypy examples/
Success: no issues found in 7 source files
(.venv) vscode ➜ /workspaces/pytradfri (enable-pre-commit) $ mypy tests/
Success: no issues found in 19 source files
```